lib/modules/login/login_router.dart

```diff
@@ -1,5 +1,18 @@
 part of 'login_module.dart';
 
-abstract class LoginRouter extends Router {}
+abstract class LoginRouter extends Router {
+  void replaceToHomeScreen(BuildContext context);
+  void pushToForgotPasswordScreen(BuildContext context);
```

Comment:
minor: usually we add a space between interface/abstract functions. I think you did this somewhere else too.